packages/neovim/tsconfig.json

```diff
@@ -4,5 +4,5 @@
     "outDir": "./lib"
   },
   "include": ["src"],
-  "exclude": ["node_modules", "__tests__", "**/*.test.ts", "examples", "lib"]
+  "exclude": ["node_modules", "examples", "lib"]
```

Comment:
to unblock this PR so we can work incrementally, maybe leave the old exclude and add specific `include` for the fixed files, e.g.:

    "include": ["src", "**/api/Buffer.test.ts"],

Reply:
Fixed